static/css/community.css

```diff
@@ -422,4 +420,11 @@ body.cid-community #cncf-code-of-conduct h2:after {
   body.cid-community .community-section#meetups p:last-of-type {
    margin-bottom: 6em; /* extra space for background */
   }
+}
+
+@media only screen and (max-width: 767px) {
+  body.cid-community .community-section h2:before,
+  body.cid-community .community-section h2:after {
+    display: none;
+  }
```

Comment:
BTW, this would remove the line through. How does the other reviewers think? If removal of the line through is acceptable, should we consider to remove them all for a cleaner view of the page? (might be too subjective)

Before:

<img width="368" alt="image" src="https://github.com/kubernetes/website/assets/11081491/5e333c79-6df5-406f-a2d7-7dc9f41c75e7">

After:

<img width="368" alt="image" src="https://github.com/kubernetes/website/assets/11081491/d7ee6c86-15be-4ccf-bbd8-3ae2f124ce62">

Reply:
It’s my decision to remove them; on mobile they are unnecessary for me, and in general they are not needed on desktop in my opinion, but I won’t change the design there. This should be discussed in a separate issue.

My task was to remove the bugs, which I did.